### ISSUE TITLE
Testcontainers for Java: Backlog

### ISSUE THREAD

**Issue by amotl** [crate/cratedb-examples#55]
Hi.

At GH-54, we are bringing in some ready-to-run code examples for basic use of _Testcontainers for Java_ with _CrateDB_. There are some items which should be addressed within subsequent iterations.

With kind regards,
Andreas.

#### Startup
- [Propagating command-line options to CrateDB](https://github.com/crate/cratedb-examples/pull/54#pullrequestreview-1381412560)

#### Database provisioning
- [Using a classpath init script](https://www.testcontainers.org/modules/databases/jdbc/#using-a-classpath-init-script)
- [Using an init script from a file](https://www.testcontainers.org/modules/databases/jdbc/#using-an-init-script-from-a-file)
- [Using an init function](https://www.testcontainers.org/modules/databases/jdbc/#using-an-init-function)

#### Other test frameworks
_Testcontainers for Java_ also provides integrations for other test frameworks. Currently, all test cases are based on JUnit 4.

- [JUnit 4](https://www.testcontainers.org/test_framework_integration/junit_4/)
- [Jupiter / JUnit 5](https://www.testcontainers.org/test_framework_integration/junit_5/)
- [Spock](https://www.testcontainers.org/test_framework_integration/spock/)

**Comment by amotl** [comment]
I think this issue can safely be closed. @florinutz: As I can see you have interest in improving testing with CrateDB with testcontainers on and off, please re-open at your disposal if you find backlog items that are worth to process.

**Comment by florinutz** [comment]
> I think this issue can safely be closed. [@florinutz](https://github.com/florinutz): As I can see you have interest in improving testing with CrateDB with testcontainers on and off, please re-open at your disposal if you find backlog items that are worth to process.

yup, I'm planning to go ahead with at least go and python for now
